faq-bundle/src/Resources/contao/modules/ModuleFaqPage.php

```diff
@@ -101,19 +103,17 @@ protected function compile()
 			$objTemp->addBefore = false;
 
 			// Add an image
-			if ($objFaq->addImage && $objFaq->singleSRC)
+			if ($objFaq->addImage && null !== ($figureBuilder = $this->getFigureBuilderIfResourceExists($objFaq->singleSRC)))
 			{
-				$objModel = FilesModel::findByUuid($objFaq->singleSRC);
-
-				if ($objModel !== null && is_file($projectDir . '/' . $objModel->path))
-				{
-					// Do not override the field now that we have a model registry (see #6303)
-					$arrFaq = $objFaq->row();
-					$arrFaq['singleSRC'] = $objModel->path;
-					$strLightboxId = 'lightbox[' . substr(md5('mod_faqpage_' . $objFaq->id), 0, 6) . ']'; // see #5810
-
-					$this->addImageToTemplate($objTemp, $arrFaq, null, $strLightboxId, $objModel);
-				}
+				$figureBuilder
+					->setSize($objFaq->size)
+					->setMetaData($objFaq->getOverwriteMetaData())
+					->setLightboxGroupIdentifier('lightbox[' . substr(md5('mod_faqpage_' . $objFaq->id), 0, 6) . ']')
+					->enableLightbox($objFaq->fullsize)
+					->build()
+					->applyLegacyTemplateData($objTemp, $objFaq->imagemargin, $objFaq->floating);
+
+				$this->Template->useImage = true;
```

Comment:
This line seems unrelated here. Maybe a copy & paste error?

Reply:
I was wondering about this the last time I visited the place but was sure this must have a reason... Turns out I cannot find one. :smile: 

Removed in d70f0466606df10fdfff2f1da509aef761c37d4c